src/plonk.rs

```diff
@@ -367,25 +329,15 @@ impl<C: HaloCurve> Circuit<C> {
                 .iter()
                 .map(|old| old.halo_g.into())
                 .collect::<Vec<_>>(),
-            c_pis_quotient.map(|c| vec![c]).unwrap_or_default(),
+            vec![c_pis_quotient],
         ]
         .concat();
 
-        let opening_points = [
-            if output_pis {
-                (0..2 * num_public_input_gates)
-                    .step_by(2)
-                    .map(|i| self.subgroup_n[i])
-                    .collect::<Vec<_>>()
-            } else {
-                vec![]
-            },
-            vec![
-                zeta_sf,
-                zeta_sf * self.subgroup_generator_n,
-                zeta_sf * self.subgroup_generator_n.exp_usize(GRID_WIDTH),
-            ],
-        ]
+        let opening_points = [vec![
+            zeta_sf,
+            zeta_sf * self.subgroup_generator_n,
+            zeta_sf * self.subgroup_generator_n.exp_usize(GRID_WIDTH),
+        ]]
         .concat();
```

Comment:
Can just be a `vec!` now right?

Reply:
Oh yes thanks! I changed it everywhere, but this one must have slipped through.